internal/Yesod/Session/Persist/Storage.hs

```diff
@@ -51,9 +51,9 @@ persistentStorage sp@SessionPersistence {} = \case
   InsertSession session ->
     persistentStorage sp (GetSession session.key) >>= \case
       Nothing -> void $ Persist.insert $ sp.toDatabase session
-      Just old -> throwWithCallStack $ SessionAlreadyExists old session
+      Just _ -> throwWithCallStack SessionAlreadyExists
```

Comment:
Did anything specific motivate this, or just simplifying?

Reply:
Yeah, there were a couple reasons - chief among them was that since the session contains an auth secret, you don't want it in a log and therefore probably don't want it in an exception either - and the information didn't feel all that useful to see anyway.

In retrospect, I wish I had given sessions a separate identifier in addition to the secret, so that we'd have some identifier that could be logged.